pkg/subctl/cmd/deploybroker.go

```diff
@@ -98,9 +101,26 @@ var deployBroker = &cobra.Command{
 		exitOnError("The provided kubeconfig is invalid", err)
 
 		status := cli.NewStatus()
-		status.Start("Deploying broker")
-		err = broker.Ensure(config)
+
+		status.Start("Setting up broker RBAC")
+		err = broker.Ensure(config, false)
```

Comment:
We discussed this in the past, and decided to forgo dealing with RBAC in the operator, because it requires granting too many privileges to the operator.

Reply:
I know, I remember, but I start to see issues with our thinking as we move the broker here too, which is different to just "provide RBAC to submariner services", ... now we need to provide RBAC / Accounts to signed up clusters...

We will need to document the RBAC steps unless we change that.

Reply:
Can we remove the `rbac.go` in favour of working with embedded yaml files?
The yaml files could be tracked in `config/rbac`. This way I can use them in the operator bundle.

Reply:
@SteveMattar this is tracked in https://github.com/submariner-io/submariner-operator/issues/1105.